src/spikeinterface/postprocessing/amplitude_scalings.py

```diff
@@ -427,13 +431,13 @@ def _are_unit_indices_overlapping(unit_inds_to_channel_indices, i, j):
     bool
         True if the unit indices i and j are overlapping, False otherwise
     """
-    if len(np.intersect1d(unit_inds_to_channel_indices[i], unit_inds_to_channel_indices[j])) > 0:
+    if np.sum(np.logical_and(sparsity_mask[i], sparsity_mask[j])) > 0:
```

Comment:
```suggestion
    if np.any(sparsity_mask[i] & sparsity_mask[j]):
```

Reply:
@zm711 : like this ?

Reply:
In my test I was still using `np.logical_and`, so `np.any(np.logical_and(x, y))` and that sped it up a little bit. I didn't try the test with the `&` operation. 

@samuelgarcia because the sparsity_mask is `(n_unit, n_chan)`, no? So it would work better with a vectorized operation?

Reply:
& in numpy is suggar syntax for np.logical_and (if I am correct), I prefer it for readability but we can go back to the previous.
sparsity_mask[i] and sparsity_mask[j] are 1d vectors.

Reply:
I think this is ok! Itàs very readable and it might be a bit faster. Good for me!